### ISSUE TITLE
Styles are not preloaded when using with nuxt build

### ISSUE THREAD

**Issue by vrtxims** [yariksav/vuetify-dialog#21]
I am using this package as a `nuxt` module `@nuxt.config.js: modules['vuetify-dialog/nuxt']`. Everything works as expected during development, but when you run the build `nuxt build -> nuxt start` the styles are not pre-loaded. But if I add the `vuetify dialog component` manually to the component where I have to call a confirmation dialog via this package the styles are fine.

![image](https://user-images.githubusercontent.com/29946299/54259095-47ced580-456d-11e9-8acc-6269b3eda13c.png)
![image](https://user-images.githubusercontent.com/29946299/54259311-fbd06080-456d-11e9-8772-296dea55e832.png)

**Comment by yariksav** [comment]
Hi, @vrtxims. Module vuetify-dialog doesn't have own styles to dialog, it use vuetify components, witch has own styles. Perhaps you use a-la-carte vuetify logic? You must install vuetify components  manually in your project (VDialog, VCard, ... and other)

**Comment by vrtxims** [comment]
Hey, @yariksav, thanks for your time, this is how `vuetify` is initialized:

```
import Vue from 'vue'
import Vuetify from 'vuetify'
import colors from 'vuetify/es5/util/colors'

Vue.use(Vuetify, {
  theme: {
    primary: '#20bf55',
    accent: '#ba1c22',
    secondary: '#ed1c24',
    info: colors.teal.lighten1,
    warning: colors.amber.base,
    error: colors.deepOrange.accent4,
    success: '#20bf55'
  }
})
```
Works in dev still, do you have any suggestion?

**Comment by vrtxims** [comment]
Importing each component separately solves the issue.

https://vuetifyjs.com/en/framework/a-la-carte

**Comment by PowerSlime** [comment]
For somebody who have this problem in latest versions of nuxt.
`"vuetify-dialog": "^1.0.0-alpha.1"`

1. Add to your nuxt.config.js `vuetify-dialog/nuxt` in your modules section.
```javascript
modules: ["vuetify-dialog/nuxt"]
```

2. Create `vuetifyDialogDependencies.js` in `plugins` folder with that inside it:
```javascript
import Vue from "vue";
import Vuetify from "vuetify";

Vue.use(Vuetify);
```

3. Add `~/plugins/vuetifyDialogDependencies` to your `plugins` section in `nuxt.config.js`
```javascript
plugins: ["~/plugins/vuetifyDialogDependencies"]
```

But the main problem here that we loading whole vuetify... Listing all of dependencies as in #30 as @baccenfutter did - didn't help me.

**Comment by PowerSlime** [comment]
Seems to be fixed in `vuetify-dialog@1.0.0-alpha.2`. Thanks @yariksav
